calendar-app/src/pages/App.tsx

```diff
@@ -0,0 +1,36 @@
+import Header from "./calendar/Header";
+import Content from "./calendar/MainContent";
+import Event from "./calendar/Event";
+import { useState } from "react";
+import { useEventContext } from "../utils/EventContext";
+function App() {
+  const [isCalendarPanel, setCalendarPanel] = useState(true);
+  const [isRightSidePanel, setRightSidePanel] = useState(true);
+  //const [isEventWindow, setEventWindow] = useState(false);
+  const { isEventWindow, setEventWindow } = useEventContext();
+  const { selectedEvent } = useEventContext();
+  return (
+    <div className="App">
+      <Header
+        calendarPanelState={isCalendarPanel}
+        setCalendarPanelDisplay={setCalendarPanel}
+      />
+      <Content
+        calendarPanelState={isCalendarPanel}
```

Comment:
React context is powerful in a way that you don't need to drill-down the props from the top. Maybe you can consume the relevant context inside the `Content` component (or deeper if needed)?